Tasks/Maksimuk/Task-1/deposit.py

```diff
@@ -0,0 +1,12 @@
+deposit = float(input("Enter initial deposit sum in BYN:\n"))
+rate = float(input("Enter interest rate:\n"))
+term = float(input("Enter term in years:\n"))
```

Comment:
The float data type tends to accumulate some error during calculations, it's better to use decimal for money-related operations, -10 points for the logical mistake